app/models/webhook_delivery.rb

```diff
@@ -0,0 +1,7 @@
+class WebhookDelivery < ApplicationRecord
+  belongs_to :course
+
+  scope :pending, -> { where.not(sent_at: nil) }
```

Comment:
This looks wrong. A delivery is pending when `sent_at` is `nil`, right?

Also, is this scope used anywhere? 🤔